kf_lib_data_ingest/validation/values.py

```diff
@@ -57,6 +55,12 @@ def inner(x):
     try_wrap(lambda x: int(x) >= 0),
 )
 
+age_units = {
```

Comment:
Kind of weird to have this here, can you move it to top of module?

Reply:
Sure thing

Reply:
moved now